dashboard/test/lib/email_reminder_test.rb

```diff
@@ -3,7 +3,7 @@
 require 'mocha/mini_test'
 
 class EmailReminderTest < ActiveSupport::TestCase
-  setup_all do
+  setup do
```

Comment:
Does switching from `setup_all` to `setup` make the data not to be shared between tests?

Reply:
Exactly. `setup_all` runs once before all the tests while `setup` runs before each individual test.